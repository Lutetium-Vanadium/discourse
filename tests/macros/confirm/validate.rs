fn main() {
    let q = requestty::questions![Confirm { validate: todo!() }];
}
