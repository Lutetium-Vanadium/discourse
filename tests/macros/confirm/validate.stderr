error: option `validate` does not exist for kind `confirm`
 --> $DIR/validate.rs:2:45
  |
2 |     let q = requestty::questions![Confirm { validate: todo!() }];
  |                                             ^^^^^^^^
